app/components/preprint-form-project-select.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+    userNodes: Ember.A(),
+    selectedNode: null,
+    isAdmin: Ember.computed('selectedNode', function() {
+        return ~this.get('selectedNode.currentUserPermissions').indexOf('admin');
```

Comment:
Though the tilde trick does have a following, it's much less readable or idiomatic than `indexOf !== -1`, so we tend to prefer the latter for clarity.

Also, rather than comparing to hard-coded strings like 'admin', we generally encourage using the constants defined in `ember-osf/const/permissions`. 

This is mainly to facilitate maintainability if we rework permissions names in the future.